### PR TITLE
Removed cross cluster monitor experimental banner, and fixed bugs.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
### Description
1. Fixed validation bug for cross cluster monitors.
2. Adding check for whether user can call an API used to configure cross cluster monitors.
3. Removed experimental banner.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js [hunk 1]
```diff
@@ -180,31 +182,55 @@ class DefineMonitor extends Component {
...
-        this.setState({ remoteMonitoringEnabled: remoteMonitoringEnabled });
+        }
       }
     } catch (e) {
-      console.log('Error while retrieving settings', e);
+      console.log('Error while retrieving settings:', e);
     }
+
+    // Check whether the user can call GetRemoteIndexes
+    if (remoteMonitoringEnabled) {
```
Comment (riysaxen-amzn) [inline public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js:217]: can we add a test to check this behaviour

**Review by riysaxen-amzn — DISMISSED** [review]

File: public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js (hunk 1 above)
Comment (AWSHurneyt) [reply public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js:217]: Will do. I have this as a fast follow-up item as I'm working on a cypress workflow that will bring up, and run on multiple connected clusters.

File: public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js [hunk 2]
```diff
@@ -180,31 +182,55 @@ class DefineMonitor extends Component {
...
+      try {
+        const query = {
+          indexes: '*,*:*',
+          include_mappings: false,
+        };
+        const response = await httpClient.get(`../api/alerting/remote/indexes`, { query: query });
+        if (response.ok) canCallGetRemoteIndexes = true;
```
Comment (amsiglan) [inline public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js:224]: rewrite as `canCallGetRemoteIndexes = response.ok`

File: public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js [hunk 3]
```diff
@@ -180,31 +182,55 @@ class DefineMonitor extends Component {
...
+      } catch (e) {
+        console.warn('Error while retrieving clusters:', e);
+      }
+    }
+
+    this.setState({
+      remoteMonitoringEnabled: remoteMonitoringEnabled,
```
Comment (amsiglan) [inline public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js:231]: no need to specify value field when it is same as key
```
{
  remoteMonitoringEnabled,
  ...
}
```

File: public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js (hunk 2 above)
Comment (AWSHurneyt) [reply public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js:224]: Revised.

File: public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js (hunk 3 above)
Comment (AWSHurneyt) [reply public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js:231]: Ah, I typically use the `remoteMonitoringEnabled: remoteMonitoringEnabled` style for clarity. I'll revise this to your suggestion.

**Review by amsiglan — APPROVED** [review]

**Review by riysaxen-amzn — APPROVED** [review]

**Pull request merged**